protocol/avdl/keybase1/identify.avdl

```diff
@@ -91,6 +91,7 @@ protocol identify {
     TeamID teamID;
     array<UserVersion> writers; // Members of the team that can write. The team OWNERs.
     map<UserVersion,IdentifyTrackBreaks> trackBreaks;
+    string folderID; // i-team TLF ID, if there is one
```

Comment:
There's a `TLFID` defined in `common.avdl`. Is there a reason not to use that?

Reply:
Sure, will do, thanks.